packages/tools/devtools/devtools-browser-extension/e2e-tests/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"compilerOptions": {
+		"target": "es2018",
+		"module": "esnext",
+		"outDir": "dist/tsc",
+		"composite": true,
+		"types": ["chai", "chrome", "mocha", "node"],
+		"jsx": "react-jsx",
+		"forceConsistentCasingInFileNames": true,
+		"skipLibCheck": true,
+	},
+	"include": ["/**/*"],
```

Comment:
I'm a bit worried about this line. It might be doing the correct thing (only looking at files within this folder), but can we make it `"./**/*"` for consistency with how we set up these test-specific `tsconfig.json` files?